### DISCUSSION TITLE
How to enable smooth scrolling on page change..?

### DISCUSSION THREAD

**Discussion by cpt-n3mo** [sveltejs/kit#9037] [Q&A] [answered]
good day,
been trying svelte, and i am liking it allot, comming from vue, it was quite easy to adjust page scrolling on route changes
is this also possible in svelte..?

As when one changes to a new route.. the pages jumps up in one shot instantly,
is there a way to make this happen somewhat smoother..

what i have been reading this should be the default but this does not seem to be the case if one creates a project in the following way.

```
npm create svelte@latest my-app
cd my-app
npm install
npm run dev
``` 

im i missing something..?

kind regards.

**Comment by teemingc** [comment, marked as answer]
Add the following rule to a global css file such as `app.css` and import it into the root layout `src/routes/+layout.svelte` with `import '../app.css'`
```css
html {
  scroll-behavior: smooth;
}
```
